02: Reading from the disk/debug.sh

```
@@ -0,0 +1 @@
+bochs -f bochs_config
```

Shellcheck (0.11.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)